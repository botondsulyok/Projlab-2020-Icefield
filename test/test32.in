field start1 0
field end1 0
neighbours start1 end1
neighbours end1 start1
field start2 0
field end2 0
neighbours start2 end2
neighbours end2 start2
entity esk1 Eskimo start1
entity pb1 icebear end1
entity esk2 Eskimo end2
entity pb2 icebear start2
stats start1
stats end1
stats start2
stats end2
stats esk1
stats pb1
stats esk2
stats pb2
move esk1 end1
stats esk1
stats pb1
move pb2 end2
stats esk2
stats pb2
stats start1
stats end1
stats start2
stats end2
stats esk1
stats pb1
stats esk2
stats pb2