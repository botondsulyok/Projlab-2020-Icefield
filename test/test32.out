id: start1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: esk1
pullers: esk1
masters: esk1
neighbours: end1

id: end1
type: iceunit
building: plains1
snow: 0
pickable: null
standers: pb1
pullers: null
masters: null
neighbours: start1

id: start2
type: iceunit
building: plains2
snow: 0
pickable: null
standers: pb2
pullers: null
masters: null
neighbours: end2

id: end2
type: iceunit
building: plains3
snow: 0
pickable: null
standers: esk2
pullers: esk2
masters: esk2
neighbours: start2

id: esk1
type: eskimo
iceUnit: start1
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: pb1
type: icebear
iceUnit: end1
movebehaviour: amph0

id: esk2
type: eskimo
iceUnit: end2
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: pb2
type: icebear
iceUnit: start2
movebehaviour: amph1

Game Over
id: esk1
type: eskimo
iceUnit: end1
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: pb1
type: icebear
iceUnit: end1
movebehaviour: amph0

id: esk2
type: eskimo
iceUnit: end2
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: pb2
type: icebear
iceUnit: end2
movebehaviour: amph1

id: start1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: end1

id: end1
type: iceunit
building: plains1
snow: 0
pickable: null
standers: pb1, esk1
pullers: esk1
masters: esk1
neighbours: start1

id: start2
type: iceunit
building: plains2
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: end2

id: end2
type: iceunit
building: plains3
snow: 0
pickable: null
standers: esk2, pb2
pullers: esk2
masters: esk2
neighbours: start2

id: esk1
type: eskimo
iceUnit: end1
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: pb1
type: icebear
iceUnit: end1
movebehaviour: amph0

id: esk2
type: eskimo
iceUnit: end2
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: pb2
type: icebear
iceUnit: end2
movebehaviour: amph1

